doc/source/scripts/taskmaster.rst

```diff
@@ -1,26 +1,26 @@
 .. scripts/taskmaster.rst
 
-taskmaster
-==========
+``taskmaster``
+==============
 
-``taskmaster`` submits a job on the PRISMS flux debug queue that will repeatedly
-resubmit any ``Auto`` jobs in the job database that have completed but whose
-taskstatus is still ``'Incomplete'`` (perhaps because the jobs has hit the walltime
-before completing or failed to converge) and then resubmit itself with a delay
-before execution.
+Summary:
+--------
 
-To use on machines other than flux change the line containing 
+``taskmaster`` submits a job that will repeatedly resubmit any ``Auto`` jobs in 
+the job database that have completed but whose taskstatus is still ``'Incomplete'`` 
+(perhaps because the jobs has hit the walltime before completing or failed to 
+converge) and then resubmit itself with a delay before execution.
 
```

Comment:
We may want a warning here that this sort of activity violates many compute cluster policies, and you should review your organization's auto-job-submission policy before proceeding with this tool.

Reply:
Is this the case? Just curious to see where. umich basically told me to do things this way. I haven't seen a policy on it at other places, but could well be ignorant of it.

Reply:
This is against policy at SNL IIRC, or at least, I've been yelled at for doing similar. I think (but haven't recently checked) that the UCSB clusters don't allow jobs to submit other jobs (so this would fail for technical, rather than regulatory, reasons).